crates/bevy_pbr/src/decal/clustered.rs

```diff
@@ -97,6 +98,72 @@ pub struct ClusteredDecal {
     pub tag: u32,
 }
 
+/// Cubemap layout defines the order of images in a packed cubemap image.
+#[derive(Default, Reflect, Debug, Clone, Copy)]
+pub enum CubemapLayout {
+    /// layout in a vertical cross format
+    ///    +y
+    /// -x -z +x
+    ///    -y
+    ///    +z
+    #[default]
+    CrossVertical = 0,
+    /// layout in a horizontal cross format
+    ///    +y
+    /// -x -z +x +z
+    ///    -y
+    CrossHorizontal = 1,
+    /// layout in a vertical sequence
+    ///   +x
+    ///   -y
+    ///   +y
+    ///   -y
+    ///   -z
+    ///   +z
+    SequenceVertical = 2,
+    /// layout in a horizontal sequence
+    /// +x -y +y -y -z +z
+    SequenceHorizontal = 3,
```

Comment:
Might want to indent your +x, -y, +y, etc. lists or surround them with \`\`\`text so that they'll render well in rustdoc.

Reply:
done